pkg/olm/clusterserviceversionlist.go

```diff
@@ -45,3 +46,30 @@ func ListClusterServiceVersion(
 
 	return csvObjects, nil
 }
+
+// ListAllInNamespaceWithNamePattern returns a cluster-wide csv inventory.
```

Comment:
Please update the comment to reflect that this function is not cluster-wide and it only return csvs that contain the name pattern